tasks/default.js

```diff
@@ -14,6 +14,7 @@ gulp.task('default', callback => {
     runSequence(
         ['copy:fonts', 'images'],
         ['css', 'scripts'],
+        ['logger:createFile'],
```

Comment:
You could always move this command up into one of the arrays above so that it is run in parallel with them? At the moment it's run once the tasks above has completed.

Reply:
Yeah I did actually have it like the below beforehand, but the clean scripts file was running after which was removing the logger file folder. The below might be wrong though so can change it if so.

```javascript
['css', 'scripts', 'logger:createFile']
```

Reply:
Looks like the [README is wrong](https://github.com/justeat/gulp-build-fozzie#logger-file) for this part.

Reply:
@kevinrodrigues In that case, it's fine to keep it as it is, it was a bit of a micro optimisation anyway 🐱

Reply:
@xander-marjoram Good spot. Updated.